examples/meta/src/regression/chaid_tree.sg.in

```diff
@@ -0,0 +1,27 @@
+File f_feats_train    = read_csv("@SHOGUN_DATA@/regression_1d_linear_features_train.dat")
+File f_feats_test     = read_csv("@SHOGUN_DATA@/regression_1d_linear_features_test.dat")
+File f_labels_train   = read_csv("@SHOGUN_DATA@/regression_1d_linear_labels_train.dat")
+
+#![create_features]
+Features feats_train = create_features(f_feats_train)
+Features feats_test = create_features(f_feats_test)
+Labels labels_train = create_labels(f_labels_train)
+#![create_features]
+
+#![set_feature_types]
+IntVector ft(1)
+ft[0] = 2
```

Comment:
@karlnapf this is causing issues. In octave this becomes a scalar value :( I wrote a fix for this but it's in another branch, not yet merged.. Also this throws an error in the meta example, but ctest doesn't pick this up (I had this issue before) and I am not sure why. The test only fails when comparing the serialised outputs in the integration test, because this will not have serialised anything because of the exception thrown when you put `ft`

Reply:
So, unless we merge your fix this meta example will fail only in Octave, right? Could it be possible to merge this PR anyway, but somehow excluding it from testing with Octave (since it is broken atm)? Just to not have to put this on hold indefinitely...

Reply:
Maybe @Hephaestus12 can just fix it here? All you need to do is replace 
https://github.com/shogun-toolbox/shogun/blob/3041ea049e9a9a2e67e32ccfa648ceab5dbad5e5/src/interfaces/swig/shogun.i#L199

with `#if defined(SWIGR) || defined(SWIGOCTAVE)`

Reply:
Yes, I'll do this.

Reply:
I hope this works :D